packages/@aws-cdk/pipelines/README.md

```diff
@@ -655,6 +655,10 @@ These command lines explained:
 > Make sure you trust all the code and dependencies that make up your CDK app.
 > Check with the appropriate department within your organization to decide on the
 > proper policy to use.
+>
+> On this note, if you have an account dedicated to CI/CD pipelines you can tailored
+> the permission granted to only allow pipelines' resources to be deployed by bootstraping 
+> it with the `--template` option with a template such as [this one](https://raw.githubusercontent.com/aws-samples/aws-bootstrap-kit-examples/main/source/1-SDLC-organization/lib/cdk-bootstrap-template.yml).
```

Comment:
The challenge we have is that CI/CD pipelines typically can create a role (for CodeBuild for example) . The role could have very permissive permission and developers can do anything in CodeBuild's ShellScriptAction. 

I'll go with making readers aware of the risk, and introduce the solution:

--- 

"If your policy includes permissions to create on attach permission to a role, developers can escalate their privilege by creating a new role with more permissive permission. Thus, we recommend to implement [permissions boundary](https://aws.amazon.com/premiumsupport/knowledge-center/iam-permission-boundaries/) in the CDK Execution role. To do this, you can bootstrap with the `--template` option with [a customized template](https://github.com/aws-samples/aws-bootstrap-kit-examples/blob/ba28a97d289128281bc9483bcba12c1793f2c27a/source/1-SDLC-organization/lib/cdk-bootstrap-template.yml#L395) that contains a permission boundary."